src/types/Movie.ts

```diff
@@ -1,7 +1,9 @@
-export interface Movie {
-  title: string;
-  description: string;
-  imgUrl: string;
-  imdbUrl: string;
-  imdbId: string;
+export class Movie {
```

Comment:
why do you rewrite interface?

Reply:
Because this way it's easier to create a new Movie
const movie = new Movie(title, description, imgUrl, imdbUrl, imdbId);

Reply:
There is no sense to use class if you have a static object.
You can create a new movie just this way:
```
const movie = { title, description, imgUrl, imdbUrl, imdbId };
```

So now your `Movie` is not a `type` but this file is in `types/` directory.